client/my-sites/subscribers/tracks/use-record-subscriber-clicked.ts

```diff
@@ -0,0 +1,17 @@
+import useRecordSubscriberTrackEvent from './use-record-subscriber-tracks-event';
+
+const useRecordSubscriberClicked = () => {
+	const recordSubscribersTracksEvent = useRecordSubscriberTrackEvent();
+
+	return (
+		where: 'title' | 'icon',
+		tracksProps: { site_id?: number | null; subscription_id?: number; user_id?: number }
+	) => {
+		recordSubscribersTracksEvent(
+			`calypso_subscribers_subscriber_${ where }_clicked`,
```

Comment:
Could you add a comment above enumerating the list of tracks event this caters for? It's for dev searchability reasons.